docs/keyless.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Keyless Mode
+description: Develop locally without requiring a Clerk account.
+---
+
+**Keyless Mode** allows you to try out Clerk in your local development environment without having to sign up for a Clerk account.
+
+> [!WARNING]
+> Keyless mode is only available in [Next.js](/docs/quickstarts/nextjs) projects when developing locally. If you attempt to build your project for production, Clerk will throw an error.
+
+## How does it work?
+
+If Clerk's SDK doesn't detect valid [Clerk keys](TODO) in your environment, it will automatically fetch a set of keys for you. These keys are associated with a Clerk development instance that has a default configuration. When running locally in Keyless Mode, you'll see a UI toast indicating that you're running in this special mode with a link to claim your instance by signing up for a Clerk account, or signing in if you already have one.
+
+Keys are written to a file in a `.clerk` directory in your project. The Clerk SDK will try to add this directory to your `.gitignore` file to prevent keys from being committed to your repository.
+
+### Default configuration
+
+Clerk development instances have the following default configuration:
+
+- **Authentication**: Email and password, Sign-in with Google, Sign-in with GitHub
+- **Verification**: Email verification is required
+- **Password reset**: Password reset is enabled
+- **MFA**: MFA is disabled
+- **Appearance**: Default appearance
+
+When using Clerk in Keyless Mode, you're not able to make configuration changes to your instance. To make changes, [claim your instance](#claiming-keys).
+
+## Keyless Mode support
+
+Keyless mode works with Clerk's SDKs for [Next.js](/docs/quickstarts/nextjs). If you're using a different framework or SDK, you'll need to manually configure Clerk keys in your environment. Visit the [Clerk Dashboard](https://dashboard.clerk.com/sign-up) to create an account, create an instance, and get your keys.
+
+## Claiming keys
+
+When you're ready to make configuration changes to your instance, you can claim the automatically generated keys. To do this, click the "Claim keys" button in the Clerk toolbar on your local application.
+
+![Claim keys modal](/docs/images/keyless/keyless-toast.png)
```

Comment:
The test in the graphic is out of date with the latest. Is it important to include this as these can easily get out of sync.

Reply:
yeah, that's fair. I felt like including screenshots made things clearer, but the maintenance burden might not be worth it. I know @victoriaxyz and @alexisintech are generally against screenshots, so I'd be fine to pull them. WDYT?

Reply:
Hihi @alexcarpenter @mwickett, I'm making changes locally to edit the guide and will remove the screenshot so don't worry about it! :)